CodedEvents/backend/migrations/0001_initial.py

```diff
@@ -0,0 +1,78 @@
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Address',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Category',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=30)),
+                ('desc', models.CharField(max_length=100)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Event',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=30)),
+                ('desc', models.CharField(max_length=100)),
+                ('address', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='backend.Address')),
+                ('categories', models.ManyToManyField(to='backend.Category')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Role',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=30)),
+                ('desc', models.CharField(max_length=100)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Ticket',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('status', models.CharField(max_length=30)),
+                ('event', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='backend.Event')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Profile',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('roles', models.ManyToManyField(to='backend.Role')),
+                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+        migrations.AddField(
+            model_name='event',
+            name='organizers',
+            field=models.ManyToManyField(to='backend.Profile'),
+        ),
+        migrations.CreateModel(
+            name='Booking',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('ticket', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='backend.Ticket')),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
```

Comment:
Shouldn't this reference the profile instead of the user? Is the profile overriding the user and are the two IDs the same?

Reply:
Good point